doc/account.md

```diff
@@ -1,8 +1,7 @@
 Account
 =======
 
-Configuration
--------------
+## Configuration
```

Comment:
Why did you make this particular change?

We should pick one format and consistently use it.  Previously we have used hyphens, so why switch to hash characters?